tests/Database/Table/Selection.whereOr().phpt

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * Test: Nette\Database\Table: WhereOr operations
+ * @dataProvider? ../databases.ini mysql
```

Comment:
Can anyone think of some function which takes at least two parameters and is implemented in all mysql, sqlite and postgesql? Otherwise I'd have to use LOCATE(), which works only on MySQL and that way the whole testcase would be skipped on anything else.